### DISCUSSION TITLE
How to change line-height in editor?

### DISCUSSION THREAD

**Discussion by qiref** [helix-editor/helix#9298] [General] [answered]
How to change line-height in editor?
I cannot find any relative info in doc or wiki. Does helix support it?

**Comment by pascalkuthe** [comment]
What is line-high supposed to mean? Please elaborate what exactly you are looking for.

**Reply by qiref** [reply]
The line-height property

**Comment by danillos** [comment, marked as answer]
You'll need to change this configuration on your terminal.
